openfecli/commands/gather.py

```diff
@@ -59,10 +200,20 @@ def legacy_get_type(res_fn):
                 type=click.Path(dir_okay=True, file_okay=False,
                                          path_type=pathlib.Path),
                 required=True)
+@click.option(
+    '--report',
+    type=click.Choice(['dg', 'ddg', 'leg'], case_sensitive=False),
```

Comment:
I'm very open to other names here, both for `--report` and for `dg/ddg/leg` (maybe `abs/rel/leg`?)

Reply:
For me, dg/ddg are very intuitive. "Leg" was less intuitive for me, but I'm not sure what might be better, how others felt about it. Maybe "legs" could be better, since then you expect more than one?

Reply:
Agreed "leg" (or even "legs") is rather undescriptive, especially to non-english speakers.

I'm not sure what a better word would be, "individual"?

Reply:
Re: uncertainties, I don't have strong feelings but I definitely would at least advise for setting the uncertainty to the same level of precision as the reported number (i.e. the amount of significant figures you truncate at indicates an implicit view of the uncertainty of your results, so 17.14 has an implicit minimum error of 0.01).